readme.md

```diff
@@ -987,9 +997,19 @@ test('rejects', async t => {
 });
 ```
 
+Because a `function` that return a `promise` is allowed to be a parameter, then you can pass `async function` that throws exception as parameter:
```

Comment:
This sentence needs to be better worded.

Reply:
maybe we can make it simple like:

> When testing an `async function` you must also wait for the assertion to complete:

@sindresorhus is it better?